### ISSUE TITLE
Slider colums dont work

### ISSUE THREAD

**Issue by laurihuusko** [dudaster/ele-custom-skin#634]
i used to have 4 column slider in desktop. Then i notices that mobile version did try to add all 4 in same screen. After changing mobile and pad columns numbers desktop also changed. So now it shows only 2 columns in desktop event settings says 4.
<img width="1543" alt="Elementor | Testi Design 2021-12-14 10-55-05" src="https://user-images.githubusercontent.com/96116125/145982204-34a00ee2-0fdf-43a6-a408-91062e366cdb.png">

**Comment by dudaster** [comment]
I'm working on a fix. I'll let you know when it is out.

**Comment by joshpears** [comment]
Awesome, I'm glad there's a fix coming. I'm having the same issue. It's like the desktop is showing the mobile column viewport and the mobile is showing the desktop. I thought it was Elementor at first since they just had an update.
![Screen Shot 2021-12-14 at 10 40 14 AM](https://user-images.githubusercontent.com/96134914/146037885-c3ef85d9-f06d-4168-8f04-da451a4ffc99.png)

**Comment by garylevansjr** [comment]
> I'm working on a fix. I'll let you know when it is out.

Do you have an  ETA? Thanks

**Comment by pbaddock** [comment]
Any update on this - it's quite a problematic situation?  Hoping for an update soon.

**Comment by samuelmente** [comment]
Would be great to have a fix soon :) Already have clients asking why their site is broken :/

**Comment by rvisima** [comment]
Really need this fixed ASAP. As with others above, I have multiple clients asking why their sites are broken.

**Comment by joshpears** [comment]
> Really need this fixed ASAP. As with others above, I have multiple clients asking why their sites are broken.

Yeah, I had to go through all my client sites and remove the feature temporarily, replacing it with just a standard grid.

**Comment by kwsim539** [comment]
While we wait for a fix, maybe we should be focusing on the cause. If it was working before, what changed that broke the feature?  This could also help isolate the issue and speed up the fix.

**Comment by Thedrelle** [comment]
Same issue here, Posting to follow. (is that a thing in Github?) 

I am currently rolling back the plugins on a site that has this feature. If I can glean more info on this I will update this comment. 

update-  the plugin worked on elementor 3.4.8, upgrade to 3.5.1 may be responsible.

**Comment by pbaddock** [comment]
Can we get an updated on progress?

**Comment by EricAxel** [comment]
Hoping for a fix for this issue soon. Thanks @Thedrelle for the rollback tip. Reverting to Elementor v 3.4.8 fixed the issue for me.

**Comment by pbaddock** [comment]
Still hitting new issues with this on various sites - any news on update timing?

**Comment by pbaddock** [comment]
Updated to the latest plugin and it looks like it's fixed now, thankyou!
